metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -1320,6 +1346,37 @@ def _create_lineage_to_upstream_tables(
                 aspect=upstream_lineage,
             )
 
+    def _create_lineage_from_unsupported_csql(
+        self, csql_urn: str, query: str, db_name: str, db_connection_type: str
+    ):
+        upstream_tables = []
+        parser = LineageRunner(query)
+        overriden_db_name = self.config.extract_lineage_from_unsupported_custom_sql_queries.database_override.get(
```

Comment:
Can you help me understand the case when `database_override` is required ?

Reply:
AFAIK, such config is not used when we convert tableau provided lineage(supported custom SQLs or directly accessed tables) to DataHub datasets.

Reply:
> AFAIK, such config is not used when we convert tableau provided lineage(supported custom SQLs or directly accessed tables) to DataHub datasets.

I believe the reason is that for customSQL tables that don't have unsupported SQL, we are extracting the correct name from `tables.fullName`. This is a (redacted) example:
```
{
  "data": {
    "customSQLTables": [
      {
        "tables": [
          {
            "fullName": "`correct-name.dataset.table`"
          }
        ],
        "database": {
          "name": "Some Name",
          "connectionType": "bigquery"
        }
      }
    ]
  }
}
```

Reply:
Thank you.